### PR TITLE
Website: Add mock UHF, mobile left nav, and other nav improvements

### LINKED ISSUE #6955: Website: Left nav is innaccessible by keyboard on component pages
### Environment Information 
- __Package version(s)__: OUIFR@6.96.0; @uifabric/fabric-website@6.7.1
- __Browser and OS versions__: All

### Please provide a reproduction of the bug ~~in a codepen~~ on the livesite:
 Go to any component page on the website (e.g. [Button](https://developer.microsoft.com/en-us/fabric#/components/button)). Attempt to tab to the left nav to filter or visit another page.

#### Actual behavior:
The left nav cannot be reached by tab navigation. You have to use page search (e.g. CTRL/CMD + 'f') to apply focus to a link in the left nav.
 
#### Expected behavior:
The left nav can be reached by tab like on the Styles pages, e.g. [Animations](https://developer.microsoft.com/en-us/fabric#/styles/animations).

#### Priorities and help requested:
Are you willing to submit a PR to fix? Yes! See #6958
 
Requested priority: Normal
 
Products/sites affected: uifabric.io

### PR DESCRIPTION
**Pull request by Jahnp**
#### Pull request checklist
- [x] Addresses an existing issue: Fixes #6955 
- [x] Include a change request file using `$ npm run change`

#### Description of changes
The website currently does not have a persistent way to be navigated when developing locally or when viewed in the non-UHF staging and prod environments (https://uifabric-tst.azurewebsites.net and https://uifabric-prod.azurewebsites.net). It's especially bad on screens <768px, which don't even render a left nav. This PR adds a simulated UHF for global, site-wide navigation, as well a mobile version of the left nav. 

There are a few other fixes here as well:
- Hides the "Fabric Preview" message bar on screens <768px
- Cleans up some of the filtering and rendering logic of the left nav that made it inaccessible by tabbing (#6955)
- Introduces throttling on left nav resize events to improve performance
- Splits out the ["Components" page](https://developer.microsoft.com/en-us/fabric#/components) from the main entry point bundle. This page isn't really essential for initial paint, and including it in the entry point is unnecessary bloat.

Note that the addition of the new top nav and mobile left nav will **not** appear on the live site. Those will only take effect when developing locally and in the aforementioned non-UHF environments.

### Before
Taken from http://uifabric-tst.azurewebsites.net/#/components/button 

#### Resting state@768x768:
image

#### Resting state@425x768:
image

### After
Taken from this PR: http://localhost:4321/#/components/button
### Resting state@768x768:
image

#### Resting state@425x768
image

#### Top Nav opened@425x768
image

#### Left Nav opened@425x768
image

### CODE REVIEW

File: apps/fabric-website/src/components/App/App.tsx [hunk 1]
```diff
@@ -72,15 +76,22 @@ export class App extends React.Component<IAppProps, any> {
...
-    const viewPortHeight = window.innerHeight;
-    isAttached = AttachedScrollUtility.shouldComponentAttach(isAttached, this._attachedScrollThreshold);
-    navHeight = this._calculateNavHeight(viewPortHeight, this._appContentRect, navHeight);
-    this.setState({
-      isAttached: isAttached,
-      navHeight: navHeight
-    });
+    clearTimeout(resizeTimer);
+
+    // Throttle the positioning/resize events, which can impact performance if unmanaged.
```
Comment (cliffkoh) [inline apps/fabric-website/src/components/App/App.tsx]: :+1:

File: apps/fabric-website/src/components/App/App.tsx [hunk 2]
```diff
@@ -107,4 +118,120 @@ export class App extends React.Component<IAppProps, any> {
     }
     return height;
   }
+
+  // Gets the section title by comparing the first part of the current URL to
```
Comment (cliffkoh) [inline apps/fabric-website/src/components/App/App.tsx]: Nit: these really should just use jsdoc format so you actually benefit from your effort of writing comments :)

File: apps/fabric-website/src/components/Nav/Nav.tsx [hunk 3]
```diff
@@ -96,34 +96,45 @@ export class Nav extends React.Component<INavProps, INavState> {
...
-    const links = pages.filter(page => !page.hasOwnProperty('isHiddenFromMainNav')).map((page: INavPage, linkIndex: number) => {
-      if (page.isCategory && !filterState) {
-        return <span>{page.pages.map((innerPage: INavPage, innerLinkIndex) => this._renderLink(innerPage, innerLinkIndex))}</span>;
-      }
-      return page.isCategory && filterState ? this._renderCategory(page, linkIndex) : this._renderLink(page, linkIndex);
-    });
+    const links = pages
+      .filter(page => {
+        // Don't render pages that are explicitly told not to, as well as top-level pages that aren't active.
+        if (page.hasOwnProperty('isHiddenFromMainNav') || (page.isUhfLink && !_hasActiveChild(page) && !_isPageActive(page))) {
```
Comment (cliffkoh) [inline apps/fabric-website/src/components/Nav/Nav.tsx]: can be simplified to 

```ts
.filter(page => 
   !page.hasOwnProperty('isHiddenFromMainNav') && !(page.isUhfLink && !_hasActiveChild(page) || 
   _isPageActive(page))
.map(...
```

File: apps/fabric-website/src/pages/GetStarted/GetStartedPage.module.scss [hunk 4]
```diff
@@ -109,9 +112,9 @@
 }
 
 @media screen and (min-width: $uhf-screen-min-mobile) {
-  img {
-    margin-bottom: 20px;
-  }
+  // img {
```
Comment (mikewheaton) [inline apps/fabric-website/src/pages/GetStarted/GetStartedPage.module.scss]: Can remove this if it's not longer in use.

Comment (Jahnp) [reply apps/fabric-website/src/pages/GetStarted/GetStartedPage.module.scss]: Thanks! Removed.

File: apps/fabric-website/src/components/App/App.tsx (hunk 2 above)
Comment (Jahnp) [reply apps/fabric-website/src/components/App/App.tsx]: Sounds reasonable to me!

File: apps/fabric-website/src/components/Nav/Nav.tsx (hunk 3 above)
Comment (Jahnp) [reply apps/fabric-website/src/components/Nav/Nav.tsx]: That's fair. I personally find one-liners like this a little unintuitive to scan, but I suppose that's what the comment is for.

**Comment by Jahnp** [comment]
Bundlesize check seems stuck pending, but this PR shouldn't affect that result. I'm going to go ahead and merge as everything else has been addressed.

**Pull request merged**